ZenPacks/zenoss/WSMAN/tests/testWSMANDataSource.py

```diff
@@ -0,0 +1,78 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+from twisted.python.failure import Failure
+
+from Products.ZenEvents import ZenEventClasses
+from Products.ZenTestCase.BaseTestCase import BaseTestCase
+
+from ZenPacks.zenoss.WSMAN.datasources.WSMANDataSource import \
+    WSMANDataSourcePlugin
+from ZenPacks.zenoss.WSMAN.tests.utils import load_pickle
+
+
+class TestWSMANDataSourcePlugin(BaseTestCase):
+
+    def setUp(self):
+        self.plugin = WSMANDataSourcePlugin()
+        self.results = load_pickle('success')
```

Comment:
Please rename the file to `wsman_plugin_results` and put under the following path `ZenPacks/zenoss/WSMAN/tests/data/monitoring/wsman_plugin_results.pkl.gz`.